scannerlib/configchecks/file_check.go

```diff
@@ -53,13 +56,21 @@ type FileCheckBatch struct {
 }
 
 // Exec executes the file checks batched by the FileCheckBatch.
-func (b *FileCheckBatch) Exec() (ComplianceMap, error) {
+func (b *FileCheckBatch) Exec(prvRes string) (ComplianceMap, string, error) {
```

Comment:
What does prvRes mean? Can you add a comment to the interface definition or use a different variable name?